akka-bench-jmh-dev/src/main/scala/akka/stream/InterpreterBenchmark.scala

```diff
@@ -26,34 +26,33 @@ class InterpreterBenchmark {
   @Benchmark
   @OperationsPerInvocation(100000)
   def graph_interpreter_100k_elements() {
-    val lock = new Lock()
-    lock.acquire()
-    new GraphInterpreterSpec.TestSetup() {
-      val identities = Vector.fill(numberOfIds)(new Identity[Int])
-      val source = new GraphDataSource("source", data100k)
-      val sink = new GraphDataSink[Int]("sink", data100k.size, lock)
-
-      val b = builder(identities:_*)
-        .connect(source, identities.head.in)
-        .connect(identities.last.out, sink)
-
-      for (i <- (0 until identities.size - 1)) {
-        b.connect(identities(i).out, identities(i + 1).in)
-      }
+    new GraphInterpreterSpecKit {
+      new TestSetup {
+        val identities = Vector.fill(numberOfIds)(new Identity[Int])
+        val source = new GraphDataSource("source", data100k)
+        val sink = new GraphDataSink[Int]("sink", data100k.size)
+
+        val b = builder(identities: _*)
+          .connect(source, identities.head.in)
+          .connect(identities.last.out, sink)
+
+        // FIXME: This should not be here, this is pure setup overhead
+        for (i <- (0 until identities.size - 1)) {
```

Comment:
these benchmarks must be fixed, these initialization steps should not be measured here. Even if we want to measure the initialization speed, the above is simply not part of the initialization since it is just an ugly test tool to create the GraphAssembly (i.e. it emulates a fuser)